RELEASE-NOTES.txt

```diff
@@ -1,5 +1,9 @@
 16.0
 -----
+
+15.9.2
+-----
+* [*] Block Editor: Fix for a crash that can occur when activity is null during capabilities update[https://github.com/wordpress-mobile/WordPress-Android/issues/13248]
```

Comment:
I was surprised not to see release notes for 15.9.1. I though it might be an issue with the merge, but [there are no notes in `develop` either](https://github.com/wordpress-mobile/WordPress-Android/blob/29e5120b1ee0a1a805fd5a14d745709ae89f6f75/RELEASE-NOTES.txt).